lgc/builder/BuilderImpl.h

```diff
@@ -583,6 +583,53 @@ class MatrixBuilder : virtual public BuilderImplBase {
   // Create matrix inverse operation.
   llvm::Value *CreateMatrixInverse(llvm::Value *const matrix, const llvm::Twine &instName = "") override final;
 
+  // Cooperative matrix operations.
+  // Enumate the supported binary operation
+  enum class OperatorType : unsigned {
+    IAdd = 128,
```

Comment:
If it is truly necessary, please explain the 128 with a short comment.

Reply:
Add a comment as "Align with spv::Op::OpIAdd".